.github/workflows/soak_infra.yml

```diff
@@ -5,7 +5,6 @@
 name: Soak Infra
 
 on:
-  pull_request:
```

Comment:
I could see us still running the build steps on PRs, but not pushing, to verify PRs don't break the image builds. You can see an example of this here:

https://github.com/vectordotdev/vector/blob/316089253076748428012292be39771b197fc4ff/.github/workflows/environment.yml#L32-L36

Reply:
Ah, that's an interesting idea!